lib/request.js

```diff
@@ -242,7 +242,7 @@ module.exports = {
    */
 
   get hostname() {
-    const host = this.host;
+    const { host } = this;
```

Comment:
@sdd I'm guessing doing the `if (!host) return ''` is more performant then doing `return (this.host || '').split(':')[0];` but please do correct me if I'm wrong